src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Reflection/Execution/AssemblyBinderImplementation.cs

```diff
@@ -34,7 +34,7 @@ private AssemblyBinderImplementation()
         public static AssemblyBinderImplementation Instance { get; } = new AssemblyBinderImplementation();
 
         partial void BindEcmaFilePath(string assemblyPath, ref AssemblyBindResult bindResult, ref Exception exception, ref bool? result);
-        partial void BindEcmaByteArray(byte[] rawAssembly, byte[] rawSymbolStore, ref AssemblyBindResult bindResult, ref Exception exception, ref bool? result);
+        partial void BindEcmaBytes(ReadOnlySpan<byte> rawAssembly, ReadOnlySpan<byte> rawSymbolStore, ref AssemblyBindResult bindResult, ref Exception exception, ref bool? result);
```

Comment:
For NativeAOT I had to change a call chain of methods to accept read-only spans instead of arrays, and the chain ends here, in a partial method that gets erased. I found no occurences of `BindEcma` in the repository, if there is an implementation somewhere else it needs to be changed.

Reply:
`BindEcmaBytes` is left over from experiments with interpreter and loading ECMA assemblies in NativeAOT.